packages/main/test/pages/Card.html

```diff
@@ -42,12 +42,158 @@
 			<ui5-icon name="group" slot="avatar"></ui5-icon>
 		</ui5-card-header>
 
-		<ui5-list id="myList3" separators="Inner">
+		<ui5-list id="myList3" separators="None">
 			<ui5-li icon="horizontal-bullet-chart">Template Based Segmentation</ui5-li>
 			<ui5-li icon="opportunity">Segmentation Models</ui5-li>
 		</ui5-list>
 	</ui5-card>
 
+	<ui5-card
+	id="card2a"
+	class="myCard">
+	<ui5-card-header
+		id="cardHeader2a"
+		slot="header"
+		title-text="Quick Links"
+		subtitle-text="quick links">
```

Comment:
Subtitles in general should be Title case. Exceptions might be long text or explanations:
subtitle-text="Quick Links">
or
subtitle-text="Quick Links Subtitle">